app/src/main/java/com/alexis/myanimecompanion/data/remote/models/Details.kt

```diff
@@ -0,0 +1,69 @@
+package com.alexis.myanimecompanion.data.remote.models
+
+import com.alexis.myanimecompanion.domain.Anime
+import com.squareup.moshi.Json
+import java.text.SimpleDateFormat
+import java.util.*
+
+data class Details(
+    @Json(name = "alternative_titles")
+    val alternativeTitles: AlternativeTitles = AlternativeTitles(),
+    val background: String = "",
+    @Json(name = "end_date")
+    val endDate: String = "",
+    val genres: List<Genre> = listOf(),
+    val id: Int = 0,
+    @Json(name = "main_picture")
+    val mainPicture: MainPicture = MainPicture(),
+    val mean: Double = 0.0,
+    val media_type: String = "",
+    @Json(name = "my_list_status")
+    val myListStatus: MyListStatus? = null,
+    val num_episodes: Int = 0,
+    val pictures: List<Picture> = listOf(),
+    val popularity: Int = 0,
+    val rank: Int = 0,
+    val rating: String = "",
+    val source: String = "",
+    @Json(name = "start_date")
+    val startDate: String = "",
+    @Json(name = "start_season")
+    val startSeason: StartSeason = StartSeason(),
+    val status: String = "",
+    val synopsis: String = "",
+    val title: String = ""
+)
+
+fun Details.asAnime(): Anime {
+    val genreList: List<String> = this.genres.map { it.name }
+    val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mmXXX")
```

Comment:
Never saw XXX at the end of a date format. Can you explain what it does?

Reply:
X parses timezone information in the format "+00:00" or "-00:00". 
For example, while parsing "2017-09-11T10:27:46+05:30", the pattern XXX produces "+05:30." . A single X at the end, instead of XXX, would produce "+05". 
This is the date format returned by the API.